activiti-services/activiti-services-rest-api/.gitignore

```diff
@@ -0,0 +1,13 @@
+/target
```

Comment:
Let's delete this file: all the content here is already ignored by the root .gitignore file.